peripherals/spi/spi_v3.yaml

```diff
@@ -0,0 +1,281 @@
+"SPI*":
+  CR1:
+    IOLOCK:
+      Unlocked: [0, "IO configuration unlocked"]
+      Locked: [1, "IO configuration locked"]
+    TCRCI:
+      AllZeros: [0, "All zeros TX CRC initialization pattern"]
+      AllOnes: [1, "All ones TX CRC initialization pattern"]
+    RCRCI:
```

Comment:
Looks like the RM has `TCRCINI` and `RCRCINI` instead, so maybe we should also patch the SVD and update these?

Reply:
Ack.